_includes/footer.html

```diff
@@ -2,7 +2,7 @@
   <div class="inner">
     <div class="site-footer-inside">
       <p class="site-info"> 
-      © US-RSE  •  2021 
+      © US-RSE  •  2021-{{ 'now' | date: "%Y" }}
```

Comment:
Surely a copyright claim should relate to when content was last added rather than when it was last rendered?